### PR TITLE
feat(cd): add a contribute section and script to get cd version

### PR DESCRIPTION
**Pull request by lahabana**
Signed-off-by: Charly Molter <charly.molter@konghq.com>

xref: https://github.com/kumahq/kuma/issues/4281

### CODE REVIEW

**Comment by michaelbeaumont** [comment]
"cd" feels confusing here, too much connotation with the bash `cd` IMO.

**Review by michaelbeaumont — APPROVED** [review]
Cool! LGTM other than the name "cd"

File: docs/docs/dev/contribute/introduction.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+The UI is written in Vuejs and Checkout the [repo](https://github.com/kumahq/kuma-gui/blob/master/DEVELOPER.md) to see how to contribute.
+
+## Testing unreleased versions
+
+Kuma publishes new binaries for every commit.
+There's a small script to download the latest version:
+
+```shell
+curl https://kuma.io/cd.sh | sh -
```
Comment (michaelbeaumont) [inline docs/docs/dev/contribute/introduction.md:27]: "cd" feels confusing here, too much connotation with the bash cd IMO. Maybe just "dev" or "preview"?

Comment (lahabana) [reply docs/docs/dev/contribute/introduction.md:27]: I'll use preview then

**Comment by michaelbeaumont** [comment]
These passive voice warnings are too strict, right? In "is written in Go," IMO "written" is just an adjective, not a passive participle.

**Review by michaelbeaumont — APPROVED** [review]

**Pull request merged**